apps/examples/board_specific_demo/ameba_spi_lcd_demo/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_AMEBA_SPI_LCD
+	bool "AMEBA_SPI_LCD example"
+	default n
+	select AMEBASMART_SPI_LCD
+	---help---
+		Enable AMEBA_SPI_LCD Example
```

Comment:
What is the usage of AMEBASMART_SPI_LCD config? There is no code for AMEBASMART_SPI_LCD.
If you don't have any codes for AMEBASMART_SPI_LCD, please remove this config.